internal/types/metadata.go

```diff
@@ -47,3 +48,14 @@ type ApplicationTarget struct {
 	Swarm      bool
 	Kubernetes bool
 }
+
+// Parents is a list of ParentMetadata items
+type Parents []ParentMetadata
```

Comment:
Overall, `ParentMetadata` is almost the same as `AppMetadata`. I think we could use it directly :angel: (`type Parents []AppMetadata`)